vpp/vaapipostprocess_scaler.cpp

```diff
@@ -93,6 +93,12 @@ VaapiPostProcessScaler::process(const SharedPtr<VideoFrame>& src,
     if (!src || !dest) {
         return YAMI_INVALID_PARAM;
     }
+    if(dest->crop.width & 0x01 || dest->crop.height & 0x01 ||
```

Comment:
Do not check dest->crop.width and dest->crop.height. When you do composite,it's very likely have odd resolution in dest,